From e076279d38466b974305fd38aae4a22e0bd9a90f Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 10 Aug 2015 14:05:15 +0200
Subject: [PATCH] dracut.sh: remove quotes from install_items and
 install_optional_items

Unfortunately these are lists with whitespaces.

Cherry-picked from: 3c00189988f6921c1affdd61a0f5025ee9b93bda
Resolves: #1520721
---
 dracut.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dracut.sh b/dracut.sh
index 90b75e51..3b91b5e8 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -1396,8 +1396,8 @@ if [[ $no_kernel != yes ]]; then
 fi
 
 if [[ $kernel_only != yes ]]; then
-    (( ${#install_items[@]} > 0 )) && inst_multiple "${install_items[@]}"
-    (( ${#install_optional_items[@]} > 0 )) && inst_multiple -o "${install_optional_items[@]}"
+    (( ${#install_items[@]} > 0 )) && inst_multiple ${install_items[@]}
+    (( ${#install_optional_items[@]} > 0 )) && inst_multiple -o ${install_optional_items[@]}
 
     [[ $kernel_cmdline ]] && printf "%s\n" "$kernel_cmdline" >> "${initdir}/etc/cmdline.d/01-default.conf"
 
